profiles/common/modules/features/communities/communities.module

```diff
@@ -335,55 +355,55 @@ function _communities_get_url_in_group($path, $original_path) {
 }
 
 
-
-/*
-* _to_arg function
-*/
-
-
-function group_name_to_arg($arg=NULL) {
+/**
+ * Implementation of _to_arg function.
+ */
+function group_name_to_arg($arg = NULL) {
```

Comment:
This is not a hook implementation, so this documentation is not correct. I've looked up how they do this in core and I found inspiration by looking at `user_uid_optional_to_arg()`.

```
/**
 * Returns $arg or the user ID of the current user if $arg is '%' or empty.
 */
function user_uid_optional_to_arg($arg) {
  ...
}
```

So they document what exactly is returned, but the parameter and return value are not described since it is always the same in these `_to_arg()` functions. So in this case it could be something like:

```
/**
 * Returns the sanitized group title or 'none' if there is no group context.
 */
function group_name_to_arg($arg = NULL) {
  ...
}
```